app/views/users/_following.html.erb

```diff
@@ -0,0 +1,34 @@
+<ul class="accordion" data-accordion data-allow-all-closed="true">
+
+  <% @followable_types.each do |followable_type| %>
+
+    <li class="accordion-item" data-accordion-item>
+
+      <a href="#" class="accordion-title">
+        <span class="icon">
+          <i class="icon icon-<%= entity_icon(followable_type) %>"></i>
+        </span>
+        <span class="title">
+          <strong><%= entity_title(followable_type) %></strong>
+        </span>
+      </a>
+
+      <div class="accordion-content" data-tab-content>
+        <table>
+          <tbody>
+            <% @follows.where(followable_type: followable_type).each do |follow| %>
+              <% if followable_type == "Proposal" %>
```

Comment:
Yes @bertocq while we were programming exactly your solution we thought that this abstraction forces the developer to follow entity_partial helper method to understand where to put and how to name new followable object partials. Because of that we decide to let these lines explicitly written, but if you prefer, we make this change in a second. I think both solutions has some disadvantage and in this case we prefer explicitness over reducing code lines.